sw/host/rom_ext_image_signer/vendor-dependencies.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
```

Comment:
It's not immediately clear what this script is doing. Would you mind adding a comment explaining it?